src/test/java/seedu/address/testutil/TypicalPoc.java

```diff
@@ -0,0 +1,72 @@
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.poc.Poc;
+
+/**
+ * A utility class containing a list of {@code Poc} objects to be used in tests.
+ */
+public class TypicalPoc {
+
+    public static final Poc ALICE = new PocBuilder().withName("Alice Pauline")
+            .withEmail("alice@example.com")
+            .withPhone("94351253")
+            .withTags("friends").build();
+    public static final Poc BENSON = new PocBuilder().withName("Benson Meier")
+            .withEmail("johnd@example.com").withPhone("98765432")
+            .withTags("owesMoney", "friends").build();
+    public static final Poc CARL = new PocBuilder().withName("Carl Kurz").withPhone("95352563")
+            .withEmail("heinz@example.com").build();
+    public static final Poc DANIEL = new PocBuilder().withName("Daniel Meier").withPhone("87652533")
+            .withEmail("cornelia@example.com").build();
+    public static final Poc ELLE = new PocBuilder().withName("Elle Meyer").withPhone("9482224")
+            .withEmail("werner@example.com").build();
+    public static final Poc FIONA = new PocBuilder().withName("Fiona Kunz").withPhone("9482427")
+            .withEmail("lydia@example.com").build();
+    public static final Poc GEORGE = new PocBuilder().withName("George Best").withPhone("9482442")
+            .withEmail("anna@example.com").build();
+
+    // Manually added
+    public static final Poc HOON = new PocBuilder().withName("Hoon Meier").withPhone("8482424")
+            .withEmail("stefan@example.com").build();
+    public static final Poc IDA = new PocBuilder().withName("Ida Mueller").withPhone("8482131")
+            .withEmail("hans@example.com").build();
+
+    // Manually added - Poc's details found in {@code CommandTestUtil}
+    public static final Poc AMY = new PocBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
+            .withEmail(VALID_EMAIL_AMY).withTags(VALID_TAG_FRIEND).build();
+    public static final Poc BOB = new PocBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
+            .withEmail(VALID_EMAIL_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
+            .build();
+
+    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
+
+    private TypicalPoc() {} // prevents instantiation
+
+    /**
+     * Returns an {@code AddressBook} with all the typical Pocs.
+     */
+    /* public static AddressBook getTypicalAddressBook() {
+        AddressBook ab = new AddressBook();
+        for (Poc company : getTypicalCompanies()) {
+            ab.addPoc(company);
+        }
+        return ab;
+    }*/
+
+    public static List<Poc> getTypicalCompanies() {
```

Comment:
Why is the name for this `getTypicalCompanies()`?

Reply:
Thank you for pointing this out